Moesif.Middleware/Helpers/MoesifCaptureOutgoingRequestHandler.cs

```diff
@@ -203,7 +185,7 @@ public MoesifCaptureOutgoingRequestHandler(HttpMessageHandler innerHandler, Dict
                     reqBody = ApiHelper.JsonDeserialize<object>(requestBody);
                     requestTransferEncoding = "json";
                 }
-                catch (Exception e)
+                catch (Exception)
```

Comment:
should we not print exception if debug is true? It will help in troubleshooting customer's issues. (same everywhere `Exception e` was replaced with `Exception`)